data-prepper-plugins/otel-trace-group-processor/src/test/java/com/amazon/dataprepper/plugins/processor/oteltracegroup/OTelTraceGroupProcessorConfigTests.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.plugins.processor.oteltracegroup;
+
+import com.amazon.dataprepper.model.configuration.PluginSetting;
+import com.amazon.dataprepper.plugins.sink.opensearch.ConnectionConfiguration;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.HashMap;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+
+@RunWith(MockitoJUnitRunner.class)
+public class OTelTraceGroupProcessorConfigTests {
+
+    @Mock
+    private ConnectionConfiguration connectionConfigurationMock;
+
+    @Test
+    public void testInitialize() {
+        try (MockedStatic<ConnectionConfiguration> connectionConfigurationMockedStatic = Mockito.mockStatic(ConnectionConfiguration.class)) {
+            connectionConfigurationMockedStatic.when(() -> ConnectionConfiguration.readConnectionConfiguration(any(PluginSetting.class)))
+                    .thenReturn(connectionConfigurationMock);
+            PluginSetting testPluginSetting = new PluginSetting("otel_trace_group_prepper", new HashMap<>());
```

Comment:
Please update this to `new PluginSetting("otel_trace_group_processor",, new HashMap<>());`

Reply:
Great catch!